packages/api/src/generated/schema.graphql

```diff
@@ -1,5 +1,21 @@
 directive @sanitize(allowedTags: [String], maxLength: Int, minLength: Int, pattern: String) on INPUT_FIELD_DEFINITION
 
+type AddFollowingToLibraryError {
```

Comment:
I wonder if this should be named something like `move` like `MoveFromFollowingToLibrary` -- but also not sure if it should move the item or copy the item.

Reply:
From user perspective, it probably makes more sense to be named `copy` than `move`